server/chat-plugins/chatlog.ts

```diff
@@ -1023,8 +1025,49 @@ export class RipgrepLogSearcher extends Searcher {
 	}
 }
 
-export const LogSearcher: Searcher = new (Config.chatlogreader === 'ripgrep' ? RipgrepLogSearcher : FSLogSearcher)();
+export class DatabaseLogSearcher extends Searcher {
+	database: PostgresDatabase;
+	constructor() {
+		super();
+		this.usePM = false;
+		this.database = new PostgresDatabase();
+	}
+	async findBattleLog(tier: ID, number: number) {
+		const SQL = require('sql-template-strings');
+		const results = await this.database.query(
+			SQL`SELECT log FROM battle_logs WHERE roomid = ${number} AND format = ${tier}`
```

Comment:
Another discord comment that was missed -- if `roomid` is a primary key, then `AND format =` is redundant. You need to decide whether to remove that, or to add `format` to a primary composite key